frappe/public/js/frappe/form/multi_select_dialog.js

```diff
@@ -288,44 +316,33 @@ frappe.ui.form.MultiSelectDialog = Class.extend({
 			page_length: this.page_length + 1,
 			query: this.get_query ? this.get_query().query : '',
 			as_dict: 1
-		}
+		};
 		frappe.call({
 			type: "GET",
-			method:'frappe.desk.search.search_widget',
+			method: 'frappe.desk.search.search_widget',
 			no_spinner: true,
 			args: args,
-			callback: function(r) {
+			callback: function (r) {
 				let more = 0;
 				me.results = [];
 				if (r.values.length) {
 					if (r.values.length > me.page_length) {
 						r.values.pop();
 						more = 1;
 					}
-					r.values.forEach(function(result) {
-						if(me.date_field in result) {
-							result["Date"] = result[me.date_field]
-						}
+					r.values.forEach(function (result) {
 						result.checked = 0;
 						result.parsed_date = Date.parse(result["Date"]);
```

Comment:
Previously we would sort based on Date field, it seems like you removed a few statements with respect to it but not all. I'd suggest to handle it better

Reply:
it was sorted before as per the date field which is removed. As our list view is always sorted by **modified date**. and I removed that intensely because no need for that because we are providing our filters now.

Reply:
```javascript
me.results.sort((a, b) => {
	return a.parsed_date - b.parsed_date;
}
```

Remove this code too

Reply:
done